app/src/components/account/UserMenu.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="user-menu">
-    <UserMenuAddress v-if="address" :address="address" />
+    <UserMenuAddress
+      v-if="address && $route.name !== `notifications`"
```

Comment:
Why not? :thinking: I don't get it. That way you can see what addresses those notifications are related to, no? Currently it is actually the only way

Reply:
notifications is a network agnostic page - the address only shows the last used address. 

addresses relevant for notifications are in the user menu (always).

i agree, not having the addresses visible on the page stinks. we will solve it properly soon.